test/signup.test.js

```diff
@@ -0,0 +1,59 @@
+const chai = require('chai');
+const app = require('../index');
+const request = require('supertest')(app);
+const db = require('../models/');
+const mocha = require('mocha');
+const faker = require('faker');
+
+const describe = mocha.describe;
+
+const expect = chai.expect; 
+const user =  {
+    email: faker.internet.email(),
+    username:faker.name.lastName(),
+    password: faker.internet.password(),
+};
+
+
+describe('POST requests', function() {
+    beforeEach(function(done){
+      db.User.destroy({
```

Comment:
so since this feature flows through the the api endpoint not the db, a good approach would be hitting the signup endpoint with valid details for signup first and then do the your thing